Amplify/Categories/Auth/Models/AuthProvider.swift

```diff
@@ -27,15 +29,18 @@ public enum AuthProvider {
     /// Auth provider that uses Google Sign-In
     case google
 
+    /// Auth provider that uses Twitter Sign-In
+    case twitter
+
     /// Auth provider that uses OpenID Connect Protocol
-    case oidc
+    case oidc(ProviderName)
```

Comment:
Lets discuss this with other team to make this aligned across platforms. For hostedUI also, we need the providername defined by the user. https://docs.aws.amazon.com/cognito/latest/developerguide/authorization-endpoint.html

>For SAML 2.0 and OpenID Connect (OIDC) identity providers (IdPs), use the name that you assigned to the IdP in your user pool.

Reply:
Auth POD Sync. Will bring this up.